.dockerignore

```diff
@@ -8,7 +8,7 @@ ops/
 *.py[cod]
 
 # virtualenv
-.venv
+**/.venv

```

Comment:
- Does this glob include `.venv` in the current dir?
- does uv introduces nested venvs or this is a general safegard?

Reply:
this is a general safeguard.
chatty said the **/ means any folder including the current one.